transcom/mymove
# INT B-192226 mobile home pricing db changes

## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-21124)

## Summary

A quick database migration to update the mobile home factor in the re_shipment_type_prices table.

### How to test

Run `make db_dev_fresh` then access the DB with DBeaver or psql, query the `re_shipment_type_prices` table, and ensure that the record whose `service_id` foreign key points to the 'DMHF' re_service has its factor changed to '33.51'.
